Create 04-git-command-quiz (#12)
barraponto/usegit: quizzes/04-git-command-quiz-rubric.md

# QUIZ 04 -- Simple Git Command

1. Which of these command will stage all files even if it's in the `gitignore` ?
   [ ] git add -A
   [ ] git add .

>    [x] git add \*

Reviewer: wow

   [ ] git add -u

> 2. Which command should you use to undoes the previous commit change without modify the existing history ?

Reviewer: ```suggestion
2. Which command should you use to undo the previous commit change without modifying the existing history ?
```

   [x] git revert
   [ ] git checkout
   [ ] git reset
   [ ] git restore

3. Which of these command is a real git command ?
   [ ] git gud
   [x] git blame
   [ ] git demon
   [ ] git ignore

> 4. To remove files from the working tree and from the index, you have to use :

Reviewer: ```suggestion
4. To remove files from the working tree and from the index, you have to use:
```

   [ ] git terminate
   [ ] git del
   [ ] git push -d
   [x] git rm

> 5. What did the command `git log -n 4` ?

Reviewer: ```suggestion
5. What does the command `git log -n 4`  do?
```

>    [x] It limit the number of commit to output to 4

Reviewer: ```suggestion
   [x] It limits the number of commits in git log output to 4
```

>    [ ] It skip 4 commit before starting to show the commit output

Reviewer: ```suggestion
   [ ] It skips 4 commit before starting to show the commit output
```

   [ ] It show only commits which have at least (or at most) 4 parent commits

>    [ ] This options doesn't exist

Reviewer: ```suggestion
   [ ] This option doesn't exist
```
